build-system/tasks/runtime-test/runtime-test-base.js

```diff
@@ -92,7 +92,15 @@ function updateBrowsers(config) {
       },
     })
     .set('edge', {browsers: ['Edge']})
-    .set('firefox', {browsers: ['Firefox']})
+    .set('firefox', {
+      browsers: ['FirefoxHeadless'],
```

Comment:
How about renaming FirefoxHeadless to Firefox_flags similar to https://github.com/ampproject/amphtml/blob/fd03c3688fe5e8c7d5c4c2f8f2e3d3f830314df7/build-system/tasks/runtime-test/runtime-test-base.js#L85